.github/workflows/ci.yaml

```diff
@@ -477,3 +477,49 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - run: just ci_codegen
+
+  # -------------- MSRV & specific version testing --------------
+  msrv:
+    name: MSRV & specific version testing
+    runs-on: ubuntu-latest # as of now, there is no real reason to test on multiple OSs
+    strategy:
+      matrix:
+       rust_version:
+         - nightly-2023-06-28   # pre https://github.com/rust-lang/rust/issues/113152
+         - nightly-2023-07-26   # post https://github.com/rust-lang/rust/issues/113152
+         - nightly              # latest nightly
```

Comment:
it would be great to add

* latest stable
* old stable e.g. 1.64.0

we can exclude them for places where we need nighly (e.g. web?), but it is great to support stable explicitly.

Reply:
Oops, I see you are just talking about *this* issue instead of MSRV, then yes, let me merge it